swift/test/Ice/scope/AllTests.swift

```diff
@@ -201,6 +201,36 @@ func allTests(helper: TestHelper) async throws {
         try test(cmap3["a"]!!.s == s1)
     }
 
+    do {
+        let obj = try communicator.stringToProxy("dpi:\(helper.getTestEndpoint())")!
```

Comment:
No stringToProxy followed by checkedCast in new code, please!

If you want to test ice_isA, call ice_isA.

Reply:
I was just copy-pasting from the lines above.
I removed all the checkedCast calls from this test; all of them were superfluous.

Reply:
If you don't remember the preferred syntax, check the top-level CHANGELOG-3.8.md or the greeter demo:

```swift
// Swift
let greeter = try makeProxy(
  communicator: communicator,
  proxyString: "greeter:tcp -h localhost -p 4061",
  type: GreeterPrx.self)
```

Reply:
I wasn't familiar enough with Swift to remember that we supported this there.
I updated `scope` in Swift to use `makeProxy`. The other languages were all already correct.